lib/ansible/modules/uri.py

```diff
@@ -627,6 +634,11 @@ def main():
 
     dict_headers = module.params['headers']
 
+    # Track our no_log values
+    no_log_strings = set([])
+    if module.params.get('url_password'):
+        no_log_strings.add(module.params.get('url_password'))
```

Comment:
Should we just use `module.no_log_values` to get this?  Instead of building our own here?  I mean we can copy it or whatever, but it would be easier than duplicating this behavior.

Reply:
Yes, we should.